traffic_stats/influxdb_tools/create/create_ts_databases.go

```diff
@@ -20,45 +20,81 @@ under the License.
 package main
 
 import (
-	"flag"
 	"fmt"
-	"os"
 
 	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
 )
 
-func main() {
+const (
+	cache           = "cache_stats"
+	deliveryService = "deliveryservice_stats"
+	daily           = "daily_stats"
+)
+
+func createFlags() []cli.Flag {
+	return []cli.Flag{
```

Comment:
We aren't shy about taking contributions, so breaking it up into smaller more manageable chunks is preferred. That way we can more easily get code committed rather than trying to manage different facets of a large PR.